technologies/app/rshiny/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+version: v1
+type: APP
+id: r-shiny
+label: r-shiny
+baseline: "Shiny is an R package that makes it easy to build interactive web apps straight from R"
+description: "Shiny is an R package that makes it easy to build interactive web apps straight from R"
+available: true
+icon: r-shiny
```

Comment:
This logo doesn't exist in [Saagie Design System ](https://saagie-design-system.netlify.app/v/latest/docs-pages-ui-core-atoms-icon). Either use R logo or ask Design Team to add the R shiny logo.

Reply:
Request made to the Design Team

Reply:
Meanwhile can we use the R logo or the dataviz logo instead of having the default logo?

Reply:
Yes

Reply:
Done also for README